Sources/core/objects/Interface.cpp

```diff
@@ -355,7 +386,9 @@ std::istream& operator>>(std::stringstream& file, HumanInterface& interf)
     interf.swap_.operator>>(file);
     interf.health_.operator>>(file);
     interf.lay_.operator>>(file);
+    interf.stop_pull_.operator>>(file);
     file >> interf.active_hand_;
     file >> interf.owner_;
+    file >> interf.pulling_;
```

Comment:
You have added all new variables, but the order is mixed up. Variables in `operator<<` and `operator>>` should be in the same order - because otherwise data for one variable will be loaded into another variable.